test/unit/pipeline/FetchUnitTest.cc

```diff
@@ -101,14 +104,17 @@ TEST_F(PipelineFetchUnitTest, TickStalled) {
 
 // Tests that the fetch unit will handle instructions that straddle fetch block
 // boundaries by automatically requesting the next block of data.
-TEST_F(PipelineFetchUnitTest, FetchUnaligned) {
+TEST_P(PipelineFetchUnitTest, FetchUnaligned) {
   MacroOp mOp = {uopPtr};
   ON_CALL(isa, getMaxInstructionSize()).WillByDefault(Return(insnMaxSizeBytes));
+  ON_CALL(isa, getMinInstructionSize()).WillByDefault(Return(insnMinSizeBytes));
   ON_CALL(memory, getCompletedReads()).WillByDefault(Return(completedReads));
 
-  // Set PC to 14, so there will not be enough data to start decoding
+  assert(insnMinSizeBytes > 1 && "min insn size too small to set PC correctly");
```

Comment:
Should this be an `if` rather than an assert so that it is always checked? Performance doesn't really matter here as its a test

Reply:
Updated to use EXPECT_GT